.circleci/config.yml

```diff
@@ -119,8 +119,21 @@ commands:
       - run: 
           name: Deploying to k8s cluster for service binary-academy
           command: |
-            echo $CA_CRT | base64 --decode > ca.crt
-            kubectl --server=${KUBE_SERVER} --certificate-authority=ca.crt --token=$SERVICEACCOUNT_TOKEN set image deployment/academy-binary-com academy-binary-com=${DOCKHUB_ORGANISATION}/binary-static-academy:${CIRCLE_SHA1}
+            for SERVER_ID in {1..5}
```

Comment:
Why it's 1 to 5?

Reply:
This under the assumption that we will have up to 5 clusters. We can add or remove config easily and we don't have to make any change in code.